src/axom/spin/tests/spin_implicit_grid.cpp

```diff
@@ -801,28 +823,28 @@ TYPED_TEST(ImplicitGridExecTest, get_candidates_box_vectorized)
 
     for(int i = 0; i < res[0]; ++i)
     {
-      grid.insert(BBox(SpacePt {0.025 + 0.1 * i, .05, .05},
-                       SpacePt {0.075 + 0.1 * i, .95, .95}),
-                  i);
+      BBoxes[i] = BBox {SpacePt {0.025 + 0.1 * i, .05, .05},
```

Comment:
Hmm -- this got a bit more awkward than I'd like...

Could we modify the code to add all the points to the BBox array and then call a single `insert` at the end?
(This test was checking functionality for `get_candidates` rather than `insert`, so I don't think it will change the behavior at all).